types/react-input-mask/index.d.ts

```diff
@@ -6,8 +6,11 @@
 
 import * as React from "react";
 
-declare namespace reactInputMask {
-    interface ReactInputMaskProps extends React.InputHTMLAttributes<HTMLInputElement> {
+export default ReactInputMask;
+export as namespace ReactInputMask;
```

Comment:
@vlaz, what is the goal of this change? `export as namespace ReactInputMask` is primarily used in UMD dependencies that _also_ make their exports available in the global scope as a variable named `ReactInputMask`. Is that what you mean to do here?

If the goal is to just be able to name `Props`, then you shouldn't need the `export as namespace`. Rather, you could instead do:

```ts
declare class ReactInputMask extends React.Component<ReactInputMask.Props> {
}
declare namespace ReactInputMask {
  interface Props extends React.InputHTMLAttribute<HTMLInputElement> {
    ...
  }
}
export default ReactInputMask;
```

Then you would be able to use it as follows:

```ts
import ReactInputMask from "react-input-mask";
const props: ReactInputMask.Props = { ... };
```